Signal/src/ViewControllers/AddToBlockListViewController.m

```diff
@@ -31,27 +38,28 @@ @interface AddToBlockListViewController () <CountryCodeViewControllerDelegate, U
 
 @property (nonatomic) UIButton *blockButton;
 
+@property (nonatomic) UITableView *contactsTableView;
+
 @property (nonatomic) NSString *callingCode;
 
+@property (nonatomic, readonly) OWSContactsManager *contactsManager;
+@property (nonatomic) NSArray<Contact *> *contacts;
```

Comment:
nit: maybe a custom setter with `AssertOnMainQueue` is in order given how we update it.